core/src/bin/generate.ts

```diff
@@ -74,6 +74,12 @@ Commands:
         description:
           "For batch generators, what should we use for the high-water-mark?",
       },
+      "sync-mode": {
+        required: false,
+        letter: "s",
+        description:
+          "For destination generators, how will we sync to the destination?",
+      },
```

Comment:
Same TODO here with the action `destination:create` - this should one day become required, once all the plugins are updated.